.changes/v3.4.0/708-improvements.md

```diff
@@ -0,0 +1 @@
+* Remove invalid space in the authentication string on Linux [GH-708]
```

Comment:
Don't want to be picky, but could you just be a bit more specific this note that it only affects token scripts and not the provider codebase itself. Just want to be transparent to whoever reads release notes.

ThanksĄ

Reply:
Is this more clear now to you?

Reply:
Yep, thanks!